src/main/java/me/mrCookieSlime/Slimefun/SlimefunPlugin.java

```diff
@@ -1,632 +1,634 @@
-package me.mrCookieSlime.Slimefun;
```

Comment:
Please don't reformat the entire class

Reply:
Actually I only deleted the * import and added single class imports but the github detected it as the whole file is changed.